### PR TITLE
feat: pre-define entries list with enforced filters and default filters

### PR DESCRIPTION
**Pull request by esakal**
- please check 'moderation' for implementation of default filters
- please check 'manual playlist > content > add entry' for enforced filters

NOTE:
- please  pull kaltura-ng master and build before reviewing this branch.
- in moderation you will not see the desired label in the tags, this is ok since I'm going to fix it once I support the restore of filters later this week

---
This change is [Reviewable](https://reviewable.io/reviews/kaltura/kmc-ng/346)

### CODE REVIEW

File: src/applications/content-moderation-app/entries/entries-list-holder.component.ts [hunk 1]
```diff
@@ -22,6 +24,9 @@ export class EntriesListHolderComponent implements OnDestroy {
 
   private _shouldConfirmEntryApproval = false; // TODO [kmcng] need to get such permissions from somewhere
   private _shouldConfirmEntryRejection = false; // TODO [kmcng] need to get such permissions from somewhere
+  public _defaultFilters: Partial<EntriesFilters> = {
+    'moderationStatuses': [{ value: '1', label: '1'}, { value: '5', label: '5'}]
```
Comment (esakal) [inline src/applications/content-moderation-app/entries/entries-list-holder.component.ts]: ignore the labels, I'm going to remove them from the api tomorrow

**Comment by diamond-darrell** [comment]
:lgtm:

---

Reviewed 11 of 11 files at r1.
Review status: all files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/346#-:-L2skX0611Mz5hoCli9M:bnfp4nl)*

**Comment by amirch1** [comment]
:lgtm:

---

Review status: 10 of 11 files reviewed at latest revision, 1 unresolved discussion.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/346#-:-L2t1NAnorOa9Ck9ED7S:bnfp4nl)*

**Comment by diamond-darrell** [comment]
:lgtm:

---

Reviewed 1 of 1 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/kaltura/kmc-ng/346#-:-L2t2DXPxzH9YozX49K9:bnfp4nl)*

**Pull request merged**